scripts/apiStage.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+#
+#
+# Quick check to see if a container is running
+#
+checkContainerRunning()
+{
+  local CHECKONE=$1
+  local COUNT=1
+  local DURATION=10
+  local DELAY=0.1
+
+  printf "Check container running..."
+
+  while [ $COUNT -le $DURATION ]; do
+    sleep $DELAY
+    (( COUNT++ ))
+    if [ -z "$($PODMAN ps | grep $CHECKONE)" ]; then
+      break
+    fi
+  done
+
+  if [ ! -z "$($PODMAN ps | grep $CHECKONE)" ]; then
+    printf "${GREEN}Container SUCCESS"
+    printf "\n\n${NOCOLOR}"
+  else
+    printf "${RED}Container ERROR"
+    printf "\n\n  Error: ${RED}Check container \"${CHECKONE}\""
+    printf "${NOCOLOR}\n"
+  fi
+}
+#
+#
+# Clean container tooling
+#
+cleanApi()
+{
+  echo "Cleaning everything, \$PODMAN and data..."
+  printf "${RED}\n"
+  $PODMAN system prune -f
+  printf "${GREEN}$PODMAN cleaning success.${NOCOLOR}\n"
+}
+#
+#
+# Stop all QPC containers
+#
+stopApi()
+{
+  $PODMAN stop -t 0 $($PODMAN ps --filter name="qpc*")
+}
+#
+#
+# Run stage/dev setup, used for local development against the latest API, partially working login
+#
+stageApi()
+{
+  local PORT=$1
+  local PASSWORD=$2
+  local NAME=$3
+  local CONTAINER=$4
+
+  $PODMAN stop -t 0 $NAME
+  $PODMAN rm $NAME
+  $PODMAN pull $CONTAINER
+
+  mkdir -p "${HOME}/.local/share/discovery/log"
+  mkdir -p "${HOME}/.local/share/discovery/data"
+  mkdir -p "${HOME}/.local/share/discovery/sshkeys"
+
+  if [ -z "$($PODMAN ps | grep $NAME)" ]; then
+    printf "\n"
+    echo "Starting API, this could take a minute..."
+    $PODMAN run -itd --rm \
+      -e QPC_SERVER_PASSWORD=$PASSWORD \
+      -e QPC_DBMS=sqlite \
+      -p $PORT:443 \
+      -v "${HOME}"/.local/share/discovery/log/:/var/log \
+      -v "${HOME}"/.local/share/discovery/data/:/var/data \
+      -v "${HOME}"/.local/share/discovery/sshkeys/:/sshkeys \
+      --tls-verify=false \
+      --name $NAME \
+      $CONTAINER
```

Comment:
I haven't tested this in _all_ environments, but `OSTYPE` is a standard variable set by bash and bash-like shells, and I confirmed it is set with the expected value in macOS 14.2 and RHEL9. I'm pretty sure this will work for the foreseeable future.

```suggestion
    MOUNT_ARGS=$([[ "$OSTYPE" == "darwin"* ]] && echo "" || echo ":z")

    $PODMAN run -itd --rm \
      -e QPC_SERVER_PASSWORD=$PASSWORD \
      -e QPC_DBMS=sqlite \
      -p $PORT:443 \
      -v "${HOME}"/.local/share/discovery/log/:/var/log"$MOUNT_ARGS" \
      -v "${HOME}"/.local/share/discovery/data/:/var/data"$MOUNT_ARGS" \
      -v "${HOME}"/.local/share/discovery/sshkeys/:/sshkeys"$MOUNT_ARGS" \
      --tls-verify=false \
      --name $NAME \
      $CONTAINER
```